plugins/cricket-scoreboard/cricket_logo_downloader.py

```
@@ -0,0 +1,177 @@
+
+        # Add common variations
+        if normalized == "TA&M":
+            variations.append("TAANDM.png")
+        elif normalized == "TAMU":
+            variations.append("TA&M.png")
+
+        return variations
+
+_downloader = LogoDownloader()
+
+def download_missing_logo(sport_key: str, team_id: str, team_abbr: str, logo_path: Path, logo_url: str = None) -> bool:
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)